### ISSUE TITLE
tensorflow lite: Some operations cannot be supported when convert to tflite format

### ISSUE THREAD

**Issue by wpj1530882136** [tensorflow/tensorflow#14936]
## System information
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
   No
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
   Ubuntu 14.04
- TensorFlow installed from (source or binary):
   source
- TensorFlow version (use command below):
  1.3.0
- Python version:
  2.7
- Bazel version (if compiling from source):
  0.7.0
- GCC/Compiler version (if compiling from source):
  gcc (Ubuntu 4.8.4-2ubuntu1~14.04.3) 4.8.4
- CUDA/cuDNN version:
  cuda8.0/cudnn6.0

# Issue:
When I convert pb file trained from tensorflow to tflite format, some operations cannot be supported, such as:
**Sign
Transpose
Unpack
Pack
Fill
RandomUniform
Select
ReverseSequence
ZerosLike
ArgMax**
and some operations cannot support INT64， such as Cast 、Gather
Hope this issure could be solved as soon?

**Comment by aselle** [comment]
See the supported op list:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/lite/g3doc/tf_ops_compatibility.md

Did you make sure to remove training related operations. Could you provide more details about your model? Did you train from a standard model script?

**Comment by wpj1530882136** [comment]
@aselle 
We trained a rnn model using encoder-decoder framework to implement the named entity recognition and generated the pb file from model.ckpt which produced in training steps.

The main code is as follows:
```
self.chars = tf.placeholder(tf.int32, [None, self.seq_len], name='chars')
self.chars_emb = tf.nn.embedding_lookup(self.char_emb, self.chars)
self.chars_emb = tf.transpose(self.chars_emb, [1, 0, 2])
self.chars_emb = tf.unstack(self.chars_emb, num=self.seq_len,
                                        name='chars_emb')
self.sequence_length = tf.reduce_sum(tf.sign(self.chars), axis=1)
self.encoder_outputs, self.encoder_state_fw, self.encoder_state_bw = rnn.static_bidirectional_rnn(
                self.encoder_cell_f,
                self.encoder_cell_b,
                self.chars_emb,
                dtype=tf.float32,
                sequence_length=self.sequence_length
            )
self.decoder_logits= self.decoder_unroll(self.encoder_outputs,
                                                                                self.encoder_state, self.feas_emb,
                                                                                self.num_cls)
self.logits = tf.transpose(self.decoder_logits, [1, 0, 2])
self.loss = tf.nn.softmax_cross_entropy_with_logits(logits=self.logits, labels=self.labels, name='loss')

```
So hereby，should I rewrite the realted function as what i've mentioned above in tf-lite to make my model work? If so, that will be a huge tuffing work. And is there any suggestions, cause we want to transfer our trained model to mobile devices. Thanks in advance .

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and this issue has an assignee.Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrehentz: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrehentz: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @andrehentz: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
